action.go

```diff
@@ -343,7 +343,7 @@ func (a Action) String() (str string, err error) {
 // permission is found, the default one `default` is used.
 // The first permission that is found to apply to an operation, but
 // doesn't allow the operation to run, will fail the verification globally
-func (a Action) VerifyACL(acl ACL, keyring io.Reader) (err error) {
+func (a Action) VerifyACL(acl ACL, keyring io.Reader, verifyACLs bool) (err error) {
```

Comment:
Was thinking just here, so if onlyPubkey (or whatever we decide to call it) is true, we only verify the key, but if it's false (default) the ACL code is run too